cli/src/meta/mod.rs

```diff
@@ -355,100 +361,129 @@ impl<'de> Deserialize<'de> for RainMetaDocumentV1Item {
     }
 }
 
-/// # Search Meta
 /// searches for a meta matching the given hash in given subgraphs urls
-pub async fn search(
-    hash: &str,
-    subgraphs: &Vec<String>,
-    timeout: u32,
-) -> anyhow::Result<query::MetaResponse> {
+pub async fn search(hash: &str, subgraphs: &Vec<String>) -> anyhow::Result<query::MetaResponse> {
     if !types::common::v1::HASH_PATTERN.is_match(hash) {
         return Err(anyhow::anyhow!("invalid hash"));
     }
-    let request_body = query::MetaQuery::build_query(query::meta_query::Variables {
-        hash: Some(hash.to_ascii_lowercase()),
-    });
+    let query_string = get_meta_query(hash);
     let mut promises = vec![];
     let client = Arc::new(
         Client::builder()
-            .timeout(std::time::Duration::from_secs(timeout as u64))
+            // .timeout(std::time::Duration::from_secs(timeout as u64))
```

Comment:
delete commented out code

Reply:
this should be a TODO

Reply:
when reqwest adds support for timouts for Clients